packages/codemods/src/command.ts

```diff
@@ -35,25 +37,56 @@ export const yargsParse = (passedArgs: string[]) => {
         'switches the filters from being inclusive, to exclusive.' +
         'If no filters provided, will exclude all mods by default',
     })
+    .option('config', {
+      alias: 'c',
+      type: 'boolean',
+      default: false,
+      description: 'switches modrunner to get args from the config file modConfig.json. False by default.',
+    })
     .parse(passedArgs);
 };
-
+/* Class responsible for parsing the npx command that runs codemods on a repo.
+   Can either take info from the command line or from a config file. */
 export class CommandParser {
   public parseArgs(passedIn: string[]): CommandParserResult {
     const parsed = yargsParse(passedIn);
     if (parsed.help) {
       return { shouldExit: true, modsFilter: () => true };
     }
+    /* Attempt to locate the modConfig file in the user's repo. */
+    const foundJsonFile = getModRunnerConfig();
+    let configObj: ModRunnerConfigType = { stringFilters: [], regexFilters: [], includeMods: false };
+    if (parsed.config) {
```

Comment:
I might suggest moving the contents of this if statement into a separate function call. That way the logic of dealing with a config file is kept contained.